democracy/views/comment_image.py

```diff
@@ -0,0 +1,14 @@
+from democracy.models.section import CommentImage
+from democracy.views.base import BaseImageSerializer
+from democracy.views.utils import Base64ImageField
+
+
+class CommentImageSerializer(BaseImageSerializer):
+    """
+    Serializer for comment_image creation.
+    """
+    image = Base64ImageField()
+
+    class Meta:
+        model = CommentImage
+        fields = ['title', 'image', 'id', 'width', 'height', 'caption']
```

Comment:
`url` field should be used instead of `image` when getting an image to be consistent with section images.